src/app/story/directives/hero.component.ts

```diff
@@ -20,7 +21,8 @@ import { StoryModel } from '../../shared';
       <div class="hero-info" *ngIf="story">
         <h2>{{story.title || '(Untitled)'}}</h2>
         <p *ngIf="story?.isNew">Not saved</p>
-        <p *ngIf="!story?.isNew">Last saved at {{story.updatedAt | timeago}}</p>
+        <p *ngIf="!story?.isNew && !story?.publishedAt">Last saved at {{story.updatedAt | timeago}}</p>
+        <p *ngIf="story?.publishedAt">{{publishedOnText}}</p>
```

Comment:
Hoping we can settle on this for now and make any adjustments needed after user feedback.